### PR TITLE
Removes os-dependent notion of signed urls -> now taken care of server-side

### PR DESCRIPTION
**Pull request by hoddr**

### CODE REVIEW

File: QUICKSTART.md [hunk 1]
```diff
@@ -381,7 +381,6 @@ ENV \
     JOB_ID=null \
     API_TOKEN=null \
     API_BASE_URL=null \
-    OS=null \
```
Comment (brimoor) [inline QUICKSTART.md:384]: these environment variable definitions should now be unnecessary. I removed them everywhere

File: voxel51/platform/config.py [hunk 2]
```diff
@@ -40,9 +40,3 @@
 # The environment variable that holds the base URL of the API
 #
 API_BASE_URL_ENV_VAR = "API_BASE_URL"
-
```
Comment (brimoor) [inline voxel51/platform/config.py:43]: this was now unused

File: voxel51/platform/utils.py [hunk 3]
```diff
@@ -24,18 +24,11 @@
 from requests.adapters import HTTPAdapter
 from requests.packages.urllib3.poolmanager import PoolManager
 
-try:
```
Comment (brimoor) [inline voxel51/platform/utils.py:27]: unused. caught via linter

File: voxel51/platform/utils.py [hunk 4]
```diff
@@ -140,10 +99,9 @@ def download(path_config, output_dir):
     Returns:
         the local path to the downloaded file
     '''
-    url = handle_macos_localhost(path_config.signed_url)
-    filename = etas.HTTPStorageClient.get_filename(url)
+    filename = etas.HTTPStorageClient.get_filename(path_config.signed_url)
```
Comment (brimoor) [inline voxel51/platform/utils.py:102]: there was an undefined `url` variable here that I caught via linting

**Review by brimoor — DISMISSED** [review]
Please test before merging. I fixed a bug that my linter caught that would have prevented this from working

File: tests/server/index.js [hunk 5]
```diff
@@ -237,7 +237,6 @@ const JOB_ID = uuid4();
       cmd += `--name ${JOB_ID} ` +
         `-e TASK_DESCRIPTION="${taskURL}" ` +
         `-e API_TOKEN="xxxxxxxxxxxxxxxx" ` +
-        `-e OS="${process.platform}" ` +
```
Comment (hoddr) [inline tests/server/index.js:240]: This may require some work in the test section as it doesn't have the url conversion there.

Comment (brimoor) [reply tests/server/index.js:240]: *will* require work. I believe this PR as is breaks the local test server for Mac users

**Comment by hoddr** [comment]
I added changes to support the correct modification for local mac docker work. Cannot test as I don't have a mac machine ~but will try to get someone to run with this branch.~

I asked @mattphotonman to give it a run with a new analytic later this afternoon if he has free time.

**Review by mattphotonman — APPROVED** [review]
Works for me on Mac laptop.

**Pull request merged**